Configuration/Commands.php

```diff
@@ -16,5 +16,11 @@
     ],
     'referenceindex:update' => [
         'class' => \TYPO3\CMS\Backend\Command\ReferenceIndexUpdateCommand::class
+    ],
+    'fluidprecompile:compile' => [
```

Comment:
I'd suggest `fluid:precompile`

Reply:
Fluid itself will never provide a `precompile` or `profile` command - it's outside the responsibility of the engine package, so there's no risk in using those. If anything, Fluid will at one time provide a `render` command and that's about it.